ogc_ws/src/despatcher/msg/RegularPayload.msg

```diff
@@ -11,9 +11,14 @@ uint8 vehicle_no
 uint8 airspeed
 uint16 alt
 bool armed
+uint8 mode
 uint16 groundspeed
+uint8 windspeed
```

Comment:
Remove this since we're not putting it in for the time being

Reply:
Done!